core/backend/src/IModelDb.ts

```diff
@@ -617,7 +617,7 @@ export abstract class IModelDb extends IModel {
         if (row.id !== undefined) {
           ids.add(row.id);
           if (ids.size > IModelDb.maxLimit) {
-            throw new IModelError(IModelStatus.BadRequest, "Max LIMIT exceeded in SELECT statement", Logger.logError, loggerCategory);
+            throw new IModelError(IModelStatus.BadRequest, "Max LIMIT exceeded in SELECT statement");

```

Comment:
At some point we need to have that discussion about where logging of errors should be handled.

Reply:
yes, we should stop logging when we throw. We should log when we catch.